docs/dev/ci/github_actions/docker_images.md

```diff
@@ -44,39 +44,133 @@ Here are some pros and cons of having your own container registry:
     * The registry does not automatically mirror images available on Docker Hub
     * The necessary images must be manually added to the registry
 
-The available Docker images use the following pattern for their names: `openvinogithubactions.azurecr.io/dockerhub/<image-name>:<image-tag>`.
+The available Docker images use the following pattern for their names: `openvinogithubactions.azurecr.io/<image-name>:<image-tag>`.
 
-Most of the images on the OpenVINO ACR are mirrors of the images with the same names on Docker Hub.
+Most of the images on the OpenVINO ACR are either mirrors of the images from Docker Hub or based on them.
 
 Examples:
-* `openvinogithubactions.azurecr.io/dockerhub/ubuntu:20.04` corresponds to `ubuntu:20.04` from Docker Hub
-* `openvinogithubactions.azurecr.io/dockerhub/ubuntu:22.04` corresponds to `ubuntu:22.04` from Docker Hub
-* `openvinogithubactions.azurecr.io/dockerhub/nvidia/cuda:11.8.0-runtime-ubuntu20.04` corresponds to `nvidia/cuda:11.8.0-runtime-ubuntu20.04` from Docker Hub
+* `openvinogithubactions.azurecr.io/library/ubuntu:22.04` corresponds to `ubuntu:22.04` from Docker Hub
+* `openvinogithubactions.azurecr.io/nvidia/cuda:11.8.0-runtime-ubuntu20.04` corresponds to `nvidia/cuda:11.8.0-runtime-ubuntu20.04` from Docker Hub
+
+Image mirroring is performed automatically via [ACR artifact cache rules](https://learn.microsoft.com/en-us/azure/container-registry/container-registry-artifact-cache),
+so no manual upload from a public DockerHub to OpenVINO ACR instance is required. 
+
+## Custom docker images and handle_docker action
+
+To optimize the time required to install dependencies in workflows and to make local reproduction of workflow steps easier, 
+we create custom docker images for different types of validation and use them in our workflows. 
+The dockerfiles for these images are stored in openvino repository in [`.github/dockerfiles`](./../../../../.github/dockerfiles) 
+folder. Examples of custom dockerfiles:
```

Comment:
I think the following are not the example of dockerfiles but rather the folder structure/organization of dockerfiles.

Reply:
Changed the wording